components/pipeline-service/production/base/update-tekton-config-features.yaml

```diff
@@ -0,0 +1,12 @@
+---
+- op: add
+  path: /spec/pipeline/results-from
+  # default upstream setting
+  # value: termination-message
+  value: sidecar-logs
+
+- op: add
+  path: /spec/pipeline/max-result-size
+  # default upstream setting
+  # value: 4096
+  value: 12288
```

Comment:
@lcarva is looking at the bootc pipelinerun logs to see if this size is sufficient

Reply:
Comparing what is in taskResults vs the container termination message, it looks like the result almost made it. Only the last image ref was cut off. The new size should be enough. Looking at the SBOM would be an extra guarantee that there aren't any other entries that are just not showing up, but I don't have access to it.

I think we should go ahead with this change. It won't make the size smaller. We may have to bump the size again, but that seems like a reasonable thing to do.